dask/array/tests/test_creation.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from numpy.testing import assert_array_almost_equal
+
+from dask.array import creation
+
+
+def test_linspace():
+    darr = creation.linspace(6, 49, 5)
+    nparr = np.linspace(6, 49)
+    assert_array_almost_equal(np.array(darr), nparr)
```

Comment:
Suggest using the `eq` function from `dask/array/tests/test_array.py`.  This will handle coercion to `np.ndarray`s, allclose issues, and will also ensure that dtypes are tracked properly.